datacube_wps/__init__.py

```diff
@@ -28,7 +28,7 @@
         try:
             pid = os.fork()
         except OSError as e:
-            raise Exception("%s [%d]" % (e.strerror, e.errno)) from e
+            raise Exception("%s [%d]" % (e.strerror, e.errno)) from e  # pylint: disable=broad-exception-raised
```

Comment:
This looks bad, I haven't looked at the code in this project, but raising a more general error while attaching no extra information... Yuck.